### ISSUE TITLE
Two Related Null Pointer Crashes on 4.8.1

### ISSUE THREAD

**Issue by compyman** [onfido/onfido-android-sdk#64]
#### What was the expected behaviour?
No Crash
#### What happened instead?
We got a crash report/Null pointer exception
#### Version info:
- `onfido-capture-sdk` version: com.onfido.sdk.capture:onfido-capture-sdk:4.8.1
- Android API version(s):  6.0..1, 7.1.1
- `compileSdkVersion`: 28
- `targetSdkVersion`: 28
- `minSdkVersion`: 16
- Support library version (if applicable): N/A (androidx) 

#### Steps to reproduce:
Not sure
#### Onfido Flow initialization code:
```kotlin
    fun startReport(requestCode: Int,
                    sdk_token: String,
                    applicant_id: String,) {
        /** Transitions to the Onfido activity - sdk_token & applicant_id are required (to configure the flow)
         **/
        val flowStepsWithOptions = arrayOf(FlowStep.WELCOME, FlowStep.CAPTURE_DOCUMENT, FlowStep.FINAL)
        val config = OnfidoConfig.builder()
                .withToken(sdk_token)
                .withApplicant(applicant_id)
                .withCustomFlow(flowStepsWithOptions)
                .build()
            onfido.startActivityForResult(context, requestCode, config)
        }
    }
```
### stack traces
1 Android Model: Z981, Android Version 6.0.1 
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mychime.waveremit.app/com.onfido.android.sdk.capture.ui.OnfidoActivity}: kotlin.KotlinNullPointerException
at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2416)
at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2476)
at android.app.ActivityThread.access$900 (ActivityThread.java:150)
at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1344)
at android.os.Handler.dispatchMessage (Handler.java:102)
at android.os.Looper.loop (Looper.java:171)
at android.app.ActivityThread.main (ActivityThread.java:5417)
at java.lang.reflect.Method.invoke (Unknown source)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:616)
kotlin.KotlinNullPointerException
at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent (Unknown source)
at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent$default (Unknown source)
at com.onfido.android.sdk.capture.ui.BaseActivity.attachBaseContext (Unknown source)
at android.app.Activity.attach (Activity.java:6222)
at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2350)
at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2476)
at android.app.ActivityThread.access$900 (ActivityThread.java:150)
at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1344)
at android.os.Handler.dispatchMessage (Handler.java:102)
at android.os.Looper.loop (Looper.java:171)
at android.app.ActivityThread.main (ActivityThread.java:5417)
at java.lang.reflect.Method.invoke (Unknown source)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:616)
```

Logcat 
```
06-03 21:57:55.880 27449 27449 I MultiDex: VM with version 2.1.0 has multidex support06-03 21:57:55.890 27449 27449 I MultiDex: Installing application06-03 21:57:55.890 27449 27449 I MultiDex: VM has multidex support, MultiDex support library is disabled.06-03 21:57:55.890 27449 27449 W System : ClassLoader referenced unknown path: /system/framework/IZTESecurity.jar06-03 21:57:55.890 27449 27449 W System : ClassLoader referenced unknown path: /system/framework/com.zte.ZTESecurity.jar06-03 21:57:55.890 27449 27449 I FirebaseInitProvider: FirebaseApp initialization unsuccessful06-03 21:57:55.900 27449 27449 W com.facebook.internal.Validate: FacebookActivity is not declared in the AndroidManifest.xml. If you are using the facebook-common module or dependent modules please add com.facebook.FacebookActivity to your AndroidManifest.xml file. See https://developers.facebook.com/docs/android/getting-started for more info.06-03 21:57:55.910 27449 27449 D Rollbar : Scheduling item file handler...06-03 21:57:55.930 27449 27449 W Java7Support: Unable to load JDK7 types (annotations, java.nio.file.Path): no Java7 support added06-03 21:57:55.980 27449 27449 W MixpanelAPI.RsrcReader: Can't load names for Android view ids from 'com.mychime.waveremit.app.R$id', ids by name will not be available in the events editor.06-03 21:57:56.010 27449 27478 I DpmTcmClient: RegisterTcmMonitor from: com.android.okhttp.TcmIdleTimerMonitor06-03 21:57:56.060 27449 27449 D AndroidRuntime: Shutting down VM06-03 21:57:56.060 27449 27449 D Rollbar : Handling uncaught exception...06-03 21:57:56.060 27449 27467 D Rollbar : Rollbar thread finishing.
```

3 Phone Model: REVVLPLUS C3701A, Android Version 7.1.1
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mychime.waveremit.app/com.onfido.android.sdk.capture.ui.OnfidoActivity}: kotlin.KotlinNullPointerException
at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2668)
at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2729)
at android.app.ActivityThread.-wrap12 (Unknown source)
at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1480)
at android.os.Handler.dispatchMessage (Handler.java:102)
at android.os.Looper.loop (Looper.java:154)
at android.app.ActivityThread.main (ActivityThread.java:6132)
at java.lang.reflect.Method.invoke (Unknown source)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:886)
at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:776)
kotlin.KotlinNullPointerException
at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent (Unknown source)
at com.onfido.android.sdk.capture.common.SdkController.getSdkComponent$default (Unknown source)
at com.onfido.android.sdk.capture.ui.BaseActivity.attachBaseContext (Unknown source)
at android.app.Activity.attach (Activity.java:6664)
at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2602)
at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2729)
at android.app.ActivityThread.-wrap12 (Unknown source)
at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1480)
at android.os.Handler.dispatchMessage (Handler.java:102)
at android.os.Looper.loop (Looper.java:154)
at android.app.ActivityThread.main (ActivityThread.java:6132)
at java.lang.reflect.Method.invoke (Unknown source)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:886)
at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:776)
```
log cat: 
```
06-04 07:34:31.689 12338 12338 I MultiDex: VM with version 2.1.0 has multidex support06-04 07:34:31.689 12338 12338 I MultiDex: Installing application06-04 07:34:31.689 12338 12338 I MultiDex: VM has multidex support, MultiDex support library is disabled.06-04 07:34:31.693 12338 12338 I FirebaseInitProvider: FirebaseApp initialization unsuccessful06-04 07:34:31.696 12338 12338 W com.facebook.internal.Validate: FacebookActivity is not declared in the AndroidManifest.xml. If you are using the facebook-common module or dependent modules please add com.facebook.FacebookActivity to your AndroidManifest.xml file. See https://developers.facebook.com/docs/android/getting-started for more info.06-04 07:34:31.728 12338 12338 W Java7Support: Unable to load JDK7 types (annotations, java.nio.file.Path): no Java7 support added06-04 07:34:31.783 12338 12338 W MixpanelAPI.RsrcReader: Can't load names for Android view ids from 'com.mychime.waveremit.app.R$id', ids by name will not be available in the events editor.06-04 07:34:31.796 12338 12382 D ZS_DBG_libc: getaddrinfo : hostname=[api.mixpanel.com] , process_name=[com.mychime.waveremit.app]06-04 07:34:31.798 12338 12382 D ZS_DBG_libc: getaddrinfo : hostname=[decide.mixpanel.com] , process_name=[com.mychime.waveremit.app]
--



```
### Other Info

All this was collected through rollbar, so no videos/screenshots sorry

**Comment by sergiomarquesmoura** [comment]
Hi @compyman .
Thanks for raising this and sorry for the inconvenience. The issue was identified and a fix is coming out in our next release. Will update you once it's live

**Comment by compyman** [comment]
Thank you for promptly figuring this out - do you know when the next release will be... released?>

**Comment by sergiomarquesmoura** [comment]
Hi @compyman . We just released the fix for this crash as part of our `4.9.0` release. Changelog here: https://github.com/onfido/onfido-android-sdk/blob/master/CHANGELOG.MD#490
